### PR TITLE
Internal: Animation styles are loaded twice in the Editor [ED-15602]

### PR DESCRIPTION
**Pull request by hein-obox**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Review by murzin-nikita — DISMISSED** [review]

File: core/page-assets/loader.php [hunk 1]
```diff
@@ -175,6 +180,14 @@ public function enqueue_assets(): void {
 		}
 	}
 
+	private function load_asset_in_editor( $asset ) {
```
Comment (louiswol94) [inline core/page-assets/loader.php:183]: Please add a return type here

**Review by rami-elementor — COMMENTED** [review]
Maybe use a simpler solution:

In line 109, before the return:
```php
		// TODO: Remove the 'e-animations' registration in v3.26.0 [ED-15471].
		if ( Plugin::$instance->preview->is_preview_mode() ) {
			$styles[] = [
				'e-animations' => [
					'src' => $this->get_css_assets_url( 'animations', 'assets/lib/animations/', true ),
					'version' => ELEMENTOR_VERSION,
					'dependencies' => [],
				],
			];
		}
```

File: core/page-assets/loader.php (hunk 1 above)
Comment (hein-obox) [reply core/page-assets/loader.php]: @louiswol94 Added

**Comment by hein-obox** [comment]
> ```
> Plugin::$instance->preview
> ```

@rami-elementor `Plugin::$instance->preview` is not available at that stage yet. I return `null` values at that point.

**Review by rami-elementor — DISMISSED** [review]

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**